src/agents/greedy.py

```diff
@@ -0,0 +1,23 @@
+from carcassonne_engine.models import SerializedGameWithID
+
+from . import Agent
+
+
+class GreedyAgent(Agent):
+    def __init__(self, id: int) -> None:
+        super().__init__()
+        self.ID = id
```

Comment:
This should be `id` per the naming convention for attributes used in our Python projects.
```suggestion
        self.id = id
```